monai/transforms/composables.py

```diff
@@ -116,6 +116,9 @@ def __call__(self, data):
 class LoadNiftid(MapTransform):
     """
     dictionary-based wrapper of LoadNifti, must load image and metadata together.
+    If load a list of files in 1 key, stack them together and add a new dimension as first dim,
```

Comment:
I would replace the comment with: 

Dictionary-based wrapper of LoadNifti, must load image and metadata together. If loading a list of files in one key, 
stack them together and add a new dimension as the first dimension, and use the meta data of the first image to 
represent the stacked result. Note that the affine transform of all the stacked images should be same.